src/test/regress/pg_regress_multi.pl

```diff
@@ -1102,6 +1101,13 @@ sub RunVanillaTests
                         ("--dbname", "$dbName"));
 }
 
+if ($useMitmproxy) {
+    until(system("lsof -i :$mitmPort") == 0) {
+        print("waiting: mitmproxy was not started yet\n");
+        sleep(1);
+    }
```

Comment:
nit, Maybe we should have a counter here to not wait indefinitely.

If the tests are run by:
- a user will have many lines of output to stdout.
- CI will get a timeout
- another party may get stuck indefinitely.

This may not be so important after all.

Reply:
Made this loop try at most 60 times.